### PR TITLE
Added depreciation warning

### PR DESCRIPTION
**Pull request by chookity-pokk**
Sorry I couldn't be of more help.

### CODE REVIEW

File: vds_api_client/vds_api_base.py [hunk 1]
```diff
@@ -366,6 +366,8 @@ def delete_rois_from_account(self, rois):
         rois: (list of int) or Rois
             collection of roi ids or an Rois object (filtered)
         """
+        warnings.warn("delete_rois_from_account is deprecated, delete method should be used instead",
+                      DeprecationWarning)
```
Comment (teije01) [inline vds_api_client/vds_api_base.py:370]: No problem. It's about the exercise right? Thanks for adding the message.

Still I'd like you to tell the user two additional things in the deprecation message:
- this method will be removed after 2021-01-01
- invoke the `delete` method on `self.rois` or `self.rois[item]`

**Comment by teije01** [comment]
Ignore the checks failing, those are not important now

File: vds_api_client/vds_api_base.py (hunk 1 above)
Comment (chookity-pokk) [reply vds_api_client/vds_api_base.py:370]: Alright, I added those in to the warning message. Sorry for the delay.

Comment (teije01) [reply vds_api_client/vds_api_base.py:370]: Perfect, thanks a lot

**Review by teije01 — APPROVED** [review]
Thanks for taking this up. I know you wanted to do more, but for that it's probably best to find a package that you use a lot in your daily work with the label `good-first-issue`. I still hope this was a nice exercise.

**Comment by teije01** [comment]
Relates to #26

**Pull request merged**